ivy/array/sorting.py

```diff
@@ -16,8 +16,42 @@ def argsort(
         *,
         out: Optional[ivy.Array] = None,
     ) -> ivy.Array:
-        return ivy.argsort(self._data, axis, descending, stable, out=out)
+        """
+        ivy.Array instance method variant of ivy.argsort. This method simply wraps the
+        function, and so the docstring for ivy.argsort also applies to this method
+        with minimal changes.
+        
+        Parameters
+        ----------
+        self 
+            input array.
+        axis
+            axis along which to sort. If set to ``-1``, the function must sort along the last axis. Default: ``-1``.
+        descending
+            sort order. If ``True``, the returned indices sort ``x`` in descending order (by value). If ``False``, the returned indices sort ``x`` in ascending order (by value). Default: ``False``.
+        stable
+            sort stability. If ``True``, the returned indices must maintain the relative order of ``x`` values which compare as equal. If ``False``, the returned indices may or may not maintain the relative order of ``x`` values which compare as equal (i.e., the relative order of ``x`` values which compare as equal is implementation-dependent). Default: ``True``.
```

Comment:
this line is too long, please update it so that one line only has up to 88 characters, rest of the content would go to a new line.